serhii.trokhymets/Home_work 3/task_1.rb

```diff
@@ -0,0 +1,36 @@
+class Easy_number
+
```

Comment:
@SerhiiTrokhymets Hi! No need to add blank lines after all objects or lines with code. Read more[ here](https://github.com/rubocop-hq/ruby-style-guide#empty-lines-around-bodies)